.github/actions/create-timestamp/action.yml

```diff
@@ -0,0 +1,16 @@
+﻿name: create-timestamp
+author: qwerty
+description: Create a timestamp in ISO 8601 format for use in tags (replaces ":" with "." for use in git tags).
+runs:
+  using: composite
+  steps:
+    - name: Get timestamp
+      shell: pwsh
+      run: |
+        # Get the current timestamp in the ISO 8601 format but replace ":" with "." because we can't use ":" in tags
+        $timestamp = (Get-Date -Format 'yyyy-MM-ddTHH.mm.ssZ' -AsUTC) -replace ':', '.'
```

Comment:
This is a different timestamp than what [the AI release annotation uses](https://github.com/Lombiq/GitHub-Actions/blob/144cff309e4c280f4b67a64f96c3624e23476b63/.github/actions/add-azure-application-insights-release-annotation/Add-ReleaseAnnotation.ps1#L12). As required in the issue, these should use the same timestamp, so the release annotation can be easily correlated with the tag.

Reply:
These actually do come out as pretty much the same format though (except for replacing : to .)
```
//(Get-Date).ToUniversalTime().GetDateTimeFormats('s')[0]
2024-08-07T08:56:06
//(Get-Date -Format 'yyyy-MM-ddTHH.mm.ssZ' -AsUTC) -replace ':', '.'
2024-08-07T08.56.37Z
```
Are you saying the release annotation needs to be changed to follow the same format? Also I am not familiar and have not been able to find a standard timestamp format that is allowed in git tags and is human readable (so not a unix timestamp for example) without doing any extra formatting. If you know of any I'm happy to look into it.

Reply:
That's for https://github.com/Lombiq/GitHub-Actions/pull/375#discussion_r1687302430. What I'm talking about here, is that the time this will record to the tag will be different than what the AI release annotation uses. So if I look at the AI release annotation, it'll show 11:23, while the tag will show 11:10.

Reply:
Ah gotcha, I have now changed when the timestamp is created inside of the workflow so that these times are much closer together (1 second difference).
Let me know if that is acceptable, if not the AI release annotation action would have to be reworked a bit so that it sets the timestamp in its output to be re-used in followup actions, however this would create a dependency between the AI release annotation action and this actions which I think is not desirable.
![image](https://github.com/user-attachments/assets/768fca75-a8b4-42e4-a718-8486990add33)

Successful run: https://github.com/Lombiq/Lombiq-Tenants-Core/actions/runs/10301769287/job/28514033543

Reply:
Solved it that way now, successful run:
https://github.com/Lombiq/Lombiq-Tenants-Core/actions/runs/10322096354/job/28576518457